__tests__/integration/nanocontracts/authority.test.ts

```diff
@@ -415,6 +415,72 @@ describe('Authority actions blueprint test', () => {
     expect(tokenDetail5.totalSupply).toBe(1000n);
     expect(tokenDetail5.authorities.mint).toBe(true);
     expect(tokenDetail5.authorities.melt).toBe(true);
+
+    // Create a new nano contract of this blueprint with token creation and deposit
```

Comment:
question: Reading these integration tests, I'm under the impression that we don't have a test for a case when more than one utxo is needed to execute this transaction.

Should we add this case? Maybe there's some condition in there that may fail when, for example, the data outputs can be covered by a set of utxos but a large amount of token mints would need another set.

Reply:
It shouldn't change anything in the logic but I added the test: https://github.com/HathorNetwork/hathor-wallet-lib/pull/877/commits/064742fcb35da2fe55b6d1b577eb9c1ab995a058